core/lib/utils.js

```diff
@@ -1,6 +1,22 @@
 'use strict';
 
+var _ = require('lodash');
+
 module.exports.requireUncached = function (module) {
     delete require.cache[require.resolve(module)];
     return require(module);
+};
+
+module.exports.extendOptions = function () {
```

Comment:
Assign is merging only first level properties, but I need deep merge with escaped Array merge.